zokrates_core/lib/ffi.cpp

```diff
@@ -0,0 +1,10 @@
+#include "ffi.hpp"
```

Comment:
@JacobEberhardt ffi changed to have memory be allocated and freed from the Rust caller cc @dark64